test/integration/outreach/outreach.js

```diff
@@ -140,6 +140,143 @@ context('Outreach', function() {
       .should('not.be.disabled');
   });
 
+  specify('User verification - success', function() {
+    cy
+      .route({
+        url: '/api/actions/1/form',
+        response: {
+          data: {
+            id: '1',
+            type: 'forms',
+            attributes: {
+              name: 'Form Name',
+            },
+          },
+        },
+      })
+      .as('routeFormAction')
+      .routeFormActionDefinition()
+      .routeFormActionFields()
+      .visit('/outreach/1', { noWait: true, isRoot: true });
+
+    cy
+      .route({
+        method: 'POST',
+        url: '/api/outreach/verification-codes',
+        delay: 100,
+        response: {
+          data: {
+            actionId: '1',
+          },
+        },
+      })
+      .as('routeCreateVerifyCodeRequest');
+
+    cy
+      .get('.verify__heading-text')
+      .should('contain', 'Request a verification code to view this health resource.');
+
+    cy
+      .get('.verify__info-text')
+      .should('contain', 'We’ll send a text message with a verification code to the phone number 123-456-7890.');
+
+    cy
+      .get('.js-submit')
+      .click()
+      .should('be.disabled');
+
+    cy
+      .wait('@routeCreateVerifyCodeRequest');
+
+    cy
+      .get('.verify__heading-text')
+      .should('contain', 'Enter your verification code.');
+
+    cy
+      .get('.verify__info-text')
+      .should('contain', 'We sent a text message with a verification code to the phone number 123-456-7890.');
+
+    cy
+      .get('.js-resend')
+      .click();
+
+    cy
+      .get('.verify__heading-text')
+      .should('contain', 'Request a verification code to view this health resource.');
+
+    cy
+      .get('.js-submit')
+      .click()
+      .wait('@routeCreateVerifyCodeRequest');
+
+    cy
+      .get('.verify__code-fields')
+      .find('.js-input')
+      .first()
+      .type('1')
+      .next()
+      .type('2')
+      .next()
+      .type('3')
+      .next()
+      .type('4');
```

Comment:
Couldn't get the inputs to automatically move to the next one in the tests. Also couldn't get the copy-paste to work either. 

So still need to investigate that more.